NetKAN/FusTek-SharedAssets.netkan

```diff
@@ -0,0 +1,19 @@
+{
+    "spec_version"   : "1.4",
+    "identifier"     : "FusTek-SharedAssets",
+    "$kref"          : "#/ckan/github/sumghai/FusTek_SharedAssets",
+    "name"           : "FusTek Aerospace - Shared Assets",
+    "abstract"       : "Agency folders shared by multiple FusTek-branded add-ons for Kerbal Space Program ",
+    "license"        : "unknown",
+    "release_status" : "stable",
```

Comment:
FYI, you don't have to specify this, unless otherwise specified the default value is assumed to be `stable`.